person.rb

```diff
@@ -1,5 +1,12 @@
-class Person
+class Nameable
+  def correct_name
+    raise NotImplementedError
+  end
+end
```

Comment:
- [ ] <b>REQUIRED:</b> `Nameable` should be in its own file so that it can be accessed by others classes.  just make sure to use `require_relative` to be able to access it.  Kindly fix it. you would have to do something similar to this.

```rb
      require_relative 'nameable'
       
      class Person < Nameable
          ...
      end